rules/no-patronum-debug/no-patronum-debug.js

```diff
@@ -0,0 +1,142 @@
+const { createLinkToRule } = require("../../utils/create-link-to-rule");
+
+module.exports = {
+  meta: {
+    type: "suggestion",
+    docs: {
+      description: "Disallow the use of patronum `debug`",
+      category: "Quality",
+      recommended: false,
+      url: createLinkToRule("no-patronum-debug"),
+    },
+    messages: {
+      noPatronumDebug: "Unexpected patronum `debug` statement",
+      removePatronumDebug: "Remove this `debug` from patronum",
+    },
+    schema: [],
+    hasSuggestions: true,
+  },
+  create(context) {
+    const importedFromPatronum = new Map();
+    const importNodes = new Map();
+
+    return {
+      ImportDeclaration(node) {
+        extractImportFromPatronum({
+          importMap: importedFromPatronum,
+          nodeMap: importNodes,
+          node,
+        });
+      },
+      CallExpression(node) {
+        const currentMethodName = node.callee?.name ?? node.callee?.object.name;
+        const importedDebugFromPatronum = importedFromPatronum.get("debug");
+
+        if (currentMethodName !== importedDebugFromPatronum) {
+          return;
+        }
+
+        context.report({
+          messageId: "noPatronumDebug",
+          node,
+          suggest: [
+            {
+              messageId: "removePatronumDebug",
+              *fix(fixer) {
+                yield* removeDebugFromPatronum({
+                  fixer,
+                  node,
+                  context,
+                  importNodes,
+                });
+              },
+            },
+          ],
+        });
+      },
+    };
+  },
+};
+
+function extractImportFromPatronum({
```

Comment:
Could we use https://github.com/effector/eslint-plugin/blob/master/utils/extract-imported-from.js for this purpose?

Reply:
Yes, sure. I'll fix it.